README.md

```diff
@@ -96,6 +96,9 @@ Usage:
         will undo the last drawing stroke (or "-z")
     gromit-mpx --redo
         will redo the last undone drawing stroke (or "-y")
+    gromit-mpx --line <startX> <startY> <endX> <endY> <color> <thickness>
+        will draw a straight line with characteristics specified by the arguments (or "-l")
+        eg: gromit-mpx -l 200 200 400 400 #C4A7E7 6	
```

Comment:
I guess the colour needs single quoutes around it as the stuff after '#' will be treated as a comment?

Reply:
Ah right, fixed that up. I overlooked shell scripts!